test/xpu_api/random/statistics_tests/common_for_distributions.hpp

```diff
@@ -0,0 +1,476 @@
+// -*- C++ -*-
+//===----------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------===//
+
+#ifndef _ONEDPL_RANDOM_STATISTICS_TESTS_COMMON_FOR_DISTRS_H
+#define _ONEDPL_RANDOM_STATISTICS_TESTS_COMMON_FOR_DISTRS_H
+
+#include <iostream>
+#include <random>
+#include <limits>
+#include <oneapi/dpl/random>
+#include <math.h>
+
+#include "statistics_common.h"
+
+// Engine parameters
+constexpr auto a = 40014u;
+constexpr auto c = 200u;
+constexpr auto m = 2147483563u;
+constexpr auto seed = 777;
+
+template <typename T>
+using Element_type = typename oneapi::dpl::internal::type_traits_t<T>::element_type;
+
+template <typename ScalarRealType, typename Distr>
+std::enable_if_t<std::is_same_v<Distr, oneapi::dpl::exponential_distribution<typename Distr::result_type>>, int>
+statistics_check(int nsamples, const std::vector<ScalarRealType>& samples, ScalarRealType lambda)
```

Comment:
Idea for the future change: to prevent extra duplications we can use this header for philox statistics test aslo. The only difference between methods in this header and in `common_for_philox_uniform_real.hpp` is the used engine(here `oneapi::dpl::linear_congruential_engine` is used) 
@egrabovskaya , could you please create a Jira about that if you agree that this change is valid?

Reply:
I changed it within the last commit

Reply:
Perfect, thank you Kate!